CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [Unreleased]
+Nothing yet.
+
+
+## [0.7.0] - 2020-06-01
+### Added
+- `disable_basic_metrics` and `disable_enhanced_metrics` configuration options.
+
+### Changed
+- Major changes in `node_exporter`-like metrics.
```

Comment:
I think it's good to add a link what's changed in these metrics(naming and e.t.c.)

Reply:
Sure, what link should be used there?

Reply:
Maybe description from this PR https://github.com/percona/rds_exporter/pull/33?